applications/GeoMechanicsApplication/tests/test_GeoMechanicsApplication.py

```diff
@@ -92,7 +91,8 @@ def AssembleTestSuites():
                         KratosGeoMechanicsDynamicsTests,
                         KratosGeoMechanicsAbsorbingBoundaryColumnTests,
                         TestSellmeijersRule,
-                        TestElementaryGroundWaterFlow
+                        TestElementaryGroundWaterFlow,
+                        KratosGeoMechanicsTransientThermalTests
```

Comment:
Sorry I do understand the reason to move many tests for a night time; however, how can I run these tests? By know I use runkratosTests command and I did not find how to run all or night cases. Small cases are always ran with runkratosTests command.

Reply:
We have a limit of 90 seconds on the build and test at CIMNE. Our small suite should remain below that, but went quite a bit over with the thermal tests. The runKratosTests.bat uses applications\GeoMechanicsApplication\tests\test_GeoMechanicsApplication.py which accepts:
runKratosTests small ( = the default )
runKtratosTests all
runKratosTests nightly